Mathlib/Algebra/Order/Hom/Monoid.lean

```diff
@@ -78,18 +78,7 @@ structure OrderAddMonoidHom (α β : Type*) [Preorder α] [Preorder β] [AddZero
 /-- Infix notation for `OrderAddMonoidHom`. -/
 infixr:25 " →+o " => OrderAddMonoidHom
 
-section
-
-/-- `OrderAddMonoidHomClass F α β` states that `F` is a type of ordered monoid homomorphisms.
-
-You should also extend this typeclass when you extend `OrderAddMonoidHom`. -/
-class OrderAddMonoidHomClass (F α β : Type*) [Preorder α] [Preorder β]
-  [AddZeroClass α] [AddZeroClass β] [FunLike F α β] extends AddMonoidHomClass F α β : Prop where
-  /-- An `OrderAddMonoidHom` is a monotone function. -/
-  monotone (f : F) : Monotone f
-#align order_add_monoid_hom_class OrderAddMonoidHomClass
-
-end
+#noalign order_add_monoid_hom_class
```

Comment:
Can you add a note explaining what happened to the class? That is, it's not longer necessary because we just use two separate classes. It would be nice to centralize that comment somehow, maybe by grouping all the `#noaligns` in this file that occur for the same reason.

Reply:
Done.